Added summaries (#146)
pybamm-team/pybamm.org: content/training.md

---
title: Training

> summary: These recordings are from previous PyBaMM workshops and conferences and are
> meant to serve the purpose of training material for use by the PyBaMM
> community of researchers, developers, and users alike.

Reviewer: ```suggestion
summary: "Past recordings and training material for PyBaMM for researchers and users"
```
